frontend/src/main/input.tdot

```diff
@@ -1,4 +1,4 @@
-create sheet monthly_budget
+create Budget sheet monthly_budget
```

Comment:
The SSTitle is for the spreadsheet and not an individual sheet. The names for the sheets are created based on the type.

Reply:
I created SSTitle keeping in mind the Sheet name tbh. I can work on adding a new node which takes in the Spreadsheet name later tonight

Reply:
Well in my PR I've reworked it so SSTitle corresponds to SpreadSheetTitle, as we don't need the nodes to know their name, only the type. The name is inferred from type. So you don't have to worry about reworking things